src/features/class-block-audit-command.php

```diff
@@ -128,23 +141,33 @@ public function run( array $args, array $assoc_args = [] ): void {
 			],
 			$user_query_args,
 		);
+
 		$query_args = self::process_csv_arguments_to_arrays( $query_args );
 		if ( isset( $query_args['post_type'] ) && is_string( $query_args['post_type'] ) && 'any' !== $query_args['post_type'] ) {
 			$query_args['post_type'] = explode( ',', $query_args['post_type'] );
 		}
 
+		$block_names = \Mantle\Support\Helpers\mixed( get_flag_value( $assoc_args, 'block_name', '' ) )
```

Comment:
Should the Mantle package providing this function be included in `composer.json`?

Reply:
I removed this dependency. Good call.